spec/models/armor_spec.rb

```diff
@@ -246,6 +282,19 @@
           expect(armor.weight).to be_nil
           expect(armor.unit_weight).to be_nil
         end
+
+        it 'removes automatically added enchantments', :aggregate_failures do
+          armor.name = 'imperial boots of resist frost'
+          armor.magical_effects = 'this will be case insensitive'
+          armor.weight = nil
+          armor.unit_weight = nil
+
+          validate
+          armor.reload
```

Comment:
Not that it particularly matters for this test, but this will change the values back to the ones in the database. We should use `armor.enchantables_enchantments.reload` instead.